lib/slide_hub/cloud/engine/aws.rb

```diff
@@ -233,7 +236,7 @@ def self.populate_policy(base_time, access_id, secret_key, security_token, regio
             ],
           }
 
-          unless security_token.empty?
+          unless security_token.blank?
```

Comment:
__[RuboCop]__

Rails/Present: Use `if security_token.present?` instead of `unless security_token.blank?`.


<!-- Comment by [SideCI](https://sideci.com) -->